tasks/main.yml

```diff
@@ -1,7 +1,11 @@
 ---
+# Copyright (C) 2018-2020 Robert Wimmer
+# SPDX-License-Identifier: GPL-3.0-or-later
+
 - name: Gather instance facts
   setup:
 
+# Installing WireGuard [[[1
```

Comment:
What is this `[[[...` useful for (here and elsewhere)?

Reply:
That is why I make small commits to explain atomic changes: https://github.com/githubixx/ansible-role-wireguard/pull/76/commits/834967abfddedf1f9ed9b232c017b68f9321d510 Editor fold marks. The atomic commits now also easily allow me to remove this commit in case you don’t like it :)

Reply:
Well, no, I don't like them :wink: That's really confusing. There are thousands of editors out there and maybe one of them recognizes this by default. Such things really needs to be handled in the editor configuration if needed.